helpdesk_mgmt_fieldservice/views/res_partner.xml

```diff
@@ -3,13 +3,14 @@
     <record id="res_partner_form_ticket_context" model="ir.ui.view">
         <field name="name">res.partner.form.ticket.context</field>
         <field name="model">res.partner</field>
-        <field name="groups_id" eval="[(4, ref('fieldservice.group_fsm_user_own'))]" />
         <field name="inherit_id" ref="base.view_partner_form" />
         <field name="arch" type="xml">
+            <field name="service_location_id" invisible="1" />
             <button name="action_view_helpdesk_tickets" position="attributes">
                 <attribute
                     name="context"
-                >{'default_partner_id': id, 'default_fsm_location_id': service_location_id}</attribute>
+                    groups="fieldservice.group_fsm_user_own"
+                >{'default_partner_id': id}</attribute>
             </button>
```

Comment:
**question:** these changes are not really clear to me, could you give me some context and explain these?

Reply:
With migration 16.0 page, this point was mentioned:
I am not able to use groups_id in views instead need to add groups attribute in the field itself. Hence this change was introduced

 Any view with groups_id on it now has to move such groups to the elements of the view. But now you can put groups attribute in a view field instead of isolating it on a view without fear of an access error. Check it in https://github.com/odoo/odoo/pull/98551.

Reply:
Okay, looks great! But why the default location has been removed?

Reply:
Sorry for delayed response. I shouldn't have removed service_location_id while incorporating "groups" for the button. Hope the latest code looks fine